modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -65,17 +65,19 @@ def pdf_structure(pdf_version)
     def assign_metadata
       return unless %w[v2 v0].include?(api_version&.downcase)
 
-      # retain original incoming non-pii form_data in metadata since this model's form_data is eventually removed
-      self.metadata = if Flipper.enabled?(:decision_review_sc_pact_act_boolean)
-                        { form_data: { evidence_type:, potential_pact_act: }, pact: { potential_pact_act: } }
-                      else
-                        { form_data: { evidence_type: } }
-                      end
-      metadata['form_data']['benefit_type'] = benefit_type
-      metadata['central_mail_business_line'] = lob
-      metadata['potential_write_in_issue_count'] = contestable_issues.filter do |issue|
-        issue['attributes']['ratingIssueReferenceId'].blank?
-      end.count
+      self.metadata = {
+        central_mail_business_line: lob,
+        form_data: { benefit_type:, evidence_type: },
+        non_veteran_claimant: non_veteran_claimant?,
+        potential_write_in_issue_count: contestable_issues.filter do |issue|
+          issue['attributes']['ratingIssueReferenceId'].blank?
+        end.count
+      }
+
+      if Flipper.enabled?(:decision_review_sc_pact_act_boolean)
+        metadata['form_data']['potential_pact_act'] = potential_pact_act
+        metadata['pact'] = { potential_pact_act: }
+      end
```

Comment:
Just an FYI, this might conflict with @michelpmcdonald's changes in a different PR, to remove the PACT Act boolean.

Reply:
Thanks, I've rebased this now that that PR is merged :+1: